op-vis/src/App.scss

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
Is `App.css` no longer useful and can be removed with this?

Reply:
Yeah, .scss just takes over all the styling

Reply:
Could you help delete App.css? Thanks!